test/cypress/integration/group-chat-message.spec.js

```diff
@@ -1,9 +1,22 @@
 import { CHATROOM_GENERAL_NAME } from '../../../frontend/model/contracts/shared/constants.js'
 
+/**
+ * Should import from this function from '../../../frontend/model/contracts/shared/functions.js'
+ * But Cypress doesn't render files using Flowtype annotations
+ * So copied that function and use it here
+ */
```

Comment:
Good job with this comment.

Could you please also add a TODO? E.g. something like:

```js
// TODO: figure out how to import functions with type annotations in Cypress
```

Reply:
Sure. Why not?